.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: python
+sudo: false
+cache: pip
+
+python:
+  - "3.5"
+
+branches:
+  only:
+    - master
+
+before-install:
+  - pip install -U pip wheel
+
+install:
+  - pip install --no-cache-dir -U -r requirements.txt
+
+script:
+  - ./check_line_length.py
+  - chmod +x runsphinx-build
```

Comment:
there was a misunderstanding: you don't need to do this here, you need to do it in your environment and then commit it (adding the executable permission to a file marks it as a changed file in git which can be committed), do that and remove this line